### DISCUSSION TITLE
References

### DISCUSSION THREAD

**Discussion by AimanFariz** [csg-utulsa/VR-Nursing-Training#81] [General]
Papers related to this project.

**Comment by gorkmo** [comment]
DOI numbers for some articles with relevant titles published within the last five years:
10.1016/j.nedt.2023.106002
10.1016/j.nedt.2022.105550
10.1016/j.nepr.2021.103056
10.1097/CIN.0000000000000866
10.1136/archdischild-2019-317011
10.1016/j.ecns.2023.04.007
10.1007/s00371-023-03227-9

**Comment by levferno** [comment]
DOI Numbers and Titles for some articles with relevant titles published within recent time

Enhancing professional competency in clinical procedures using head-mounted display virtual reality - a mixed method study 10.1080/10872981.2023.2232134

Virtual reality simulation for nursing education: effectiveness and feasibility
10.1186/s12912-023-01639-5.

The Effect of Virtual Presence and Self-Efficacy on Nursing Students' Evaluation of Virtual Simulation Training
 10.1080/07370016.2023.2197886.
 
Analysis of cybersickness in virtual nursing simulation: a German longitudinal study
10.1186/s12912-024-01833-z

An experimental learning-based virtual reality approach to fostering problem-resolving competence in professional training
10.1080/10494820.2021

My hands are running away - learning a complex nursing skill via virtual reality simulation: a randomized mixed methods study.
10.1186/s12912-023-01384-9

Face-to-face versus 360 degrees VR video: a comparative study of two teaching methods in nursing education
 10.1186/s12912-024-01866-4.
 
 Virtual reality and Augmented Reality Training in Disaster Medicine Courses for Students in Nursing: A scoping Review of Adoptable Tools
 10.3390/bs13070616
 
 An Examination of the Effects of Virtual Reality Training on Spatial Visualization and Transfer of Learning
 10.3390/brainsci13060890
